### DISCUSSION TITLE
How can I visualize weather data of this XML format?

### DISCUSSION THREAD

**Discussion by wdma** [grafana/grafana-infinity-datasource#409] [Q&A] [answered]
I am very new to both Grafana and the Infinity data source. I have tried to follow the tutorials, but cannot scrape the relevant data.  My menus appear slightly different than in the tutorial.  Can someone please help me?

Example data are here:

https://www.nohrsc.noaa.gov/shef_archive/index.html?rfc=cnrfc&product=swe&year=2019&month=3&day=27&hour=12

The relevant data are located in this element:

body.table[3].tbody[0].tr[0].td[1].pre 

Thank you for any help you can give me!

**Comment by yesoreyeram** [comment, marked as answer]
Above data doesn't seems to be structured. ( at least data should be in table format ... not inside pre tag ).

**Reply by wdma** [reply]
Ah. Thank you. I have a similar issue (I think) with the demo, however (issue #410). I am very new to Grafana and just need an initial example to work so that I can play with it.  Thank you for your help!
